.github/PULL_REQUEST_TEMPLATE.MD

```diff
@@ -0,0 +1,25 @@
+## Describe your changes
+
+
+## Describe any testing you have performed
+
+*Please add any demo scripts to [CIL-Demos/misc/](https://github.com/TomographicImaging/CIL-Demos/tree/main/misc)*
+
+
+## Link relevant issues
+
+
+## Checklist when you are ready to request a review
+
+- [ ] I have performed a self-review of my code
+- [ ] I have added docstrings in line with the guidance in the developer guide
+- [ ] I have implemented unit tests that cover any new or modified functionality
+- [ ] CHANGELOG.md has been updated with any functionality change
+- [ ] Request review from all relevant developers
+- [ ] Change pull request label to 'Waiting for review' 
+
+## Contribution Notes
+
+Please read and adhere to the [developer guide](https://tomographicimaging.github.io/CIL/nightly/developer_guide.html) and local patterns and conventions.
+
+All contributed code must be under the [Apache2 License](https://www.apache.org/licenses/LICENSE-2.0)
```

Comment:
The SPDX short identifier is [Apache-2.0](https://spdx.org/licenses/Apache-2.0.html)

What is SPDX? I don't know, but that link is at https://www.apache.org/licenses/LICENSE-2.0.html